port/posix/posix_transport_tcp.c

```diff
@@ -21,13 +22,21 @@
 #include "wolfhsm/wh_transport.h"
 #include "port/posix/posix_transport_tcp.h"
 
+#if defined(__APPLE__) || defined(__FreeBSD__) || defined(__OpenBSD__) \
```

Comment:
Barf.  Lemme see if send(..., MSG_NO_SIGNAL) works on MacOS.  Probably does and I'm fine with using send/recv instead of read/write.  The intent of to support Posix OS's so we don't need to hit BSD directly.  I should have verified Linux handles the sig pipe first.  My bad.  More to come.